app/assets/javascripts/discourse/app/controllers/topic.js

```diff
@@ -1363,7 +1401,22 @@ export default Controller.extend(bufferedProperty("model"), {
             break;
           }
           case "created": {
-            postStream.triggerNewPostInStream(data.id).then(() => refresh());
+            this.newPostsInStream = this.newPostsInStream || [];
```

Comment:
We should also initialize this variable in the class, and probably use an underscore to mark it private.